hkm/hkm_client.py

```diff
@@ -1,85 +0,0 @@
-# -*- coding: utf-8 -*-
-
-import logging
-import urllib
-import urlparse
-from StringIO import StringIO
-
-import requests
-from PIL import Image
-
-from django.core.cache import cache
-
-
-LOG = logging.getLogger(__name__)
-
-
-class HKMClient(object):
-    timeout = 6
-
-    def get_full_res_image_url(self, preview_image_url):
```

Comment:
I'm a bit unsure about removing the whole thing here.

Aki told me that when they publish new images using the Mediaksi system, they can publish either of these:
* just the metadata about the image
* metadata + web image
* metadata + web image + full res image.

Therefore it's possible to have an image turn up as a search result so that it doesn't have a full res image available. This is probably the reason why the original solution was to do a HEAD query into the proxy server's url.

When there was no full res image available, the app didn't offer the "download full res image" or "buy" buttons in the detail page. Just last week there were 72 cases like this in production, so it's bound to happen again. We should make sure the app won't do anything funny in these cases.